misc/web-abat-xwalk-tests/tests.full.xml

```diff
@@ -3,122 +3,122 @@
   <test_definition>
     <suite category="W3C/HTML5 APIs" launcher="xwalk" name="web-abat-xwalk-tests">
       <set name="abat" type="js">
-        <testcase component="WebAPI/Media/HTML5 The audio element (Partial)" execution_type="auto" priority="P0" status="approved" platform="all" type="Functional" id="audio_canplaytype_ogg_supported_but_no_codecs_parameter" purpose="Check that audio.canPlayType method supported ogg format codecs parameter">
+        <testcase component="BAT" execution_type="auto" id="audio_canplaytype_ogg_supported_but_no_codecs_parameter" platform="all" priority="P0" purpose="Check that audio.canPlayType method supported ogg format codecs parameter" status="approved" type="Functional">
```

Comment:
Can you point me why make this update? Please address the reason in commit message.

Another, does `tests.xml` need this update?